languagetool-core/src/main/java/org/languagetool/rules/spelling/hunspell/HunspellRule.java

```diff
@@ -577,53 +575,4 @@ protected boolean isInIgnoredSet(String word) {
            // A better way might be nice.
            SpellingCheckRule.LANGUAGETOOL.equals(word) || SpellingCheckRule.LANGUAGETOOLER.equals(word);
   }
-
-  private static synchronized String getDictionaryPath(String dicName,
-                                                       String originalPath) throws IOException {
-
-    URL dictURL = JLanguageTool.getDataBroker().getFromResourceDirAsUrl(originalPath);
-    String dictionaryPath;
-    //in the webstart, java EE or OSGi bundle version, we need to copy the files outside the jar
-    //to the local temporary directory
-    if (StringUtils.equalsAny(dictURL.getProtocol(), "jar", "vfs", "bundle", "bundleresource")) {
```

Comment:
Can we have this check in your change, too? When running LT on our Servers, we have all language-module jar extracted in our file system see [this](https://github.com/languagetool-org/languagetool/blob/62d5c76427f2fe7bacf762f2af8857df69df8538/languagetool-standalone/src/main/assembly/zip.xml#L80). This means we don't need the copy to `/tmp`, before using the files.

On the contrary, it creates unnecessary files.

Sorry for not seeing this in the first review.

Reply:
Makes sense. I'd check for the `file://` protocol though, what do you think?